tasks/main.yml

```diff
@@ -1,3 +1,29 @@
 ---
 - include: setup-RedHat.yml
-  when: ansible_os_family == 'RedHat'
\ No newline at end of file
+  when: ansible_os_family == 'RedHat'
+
+- include: setup-Debian.yml
+  when: ansible_os_family == 'Debian'
+
+
+- name: Download the awscli bundle.
+  get_url: url=https://s3.amazonaws.com/aws-cli/awscli-bundle.zip dest=/tmp/awscli-bundle.zip
+
+- name: Unarchive the installer.
+  unarchive: src=/tmp/awscli-bundle.zip dest=/tmp copy=no creates=/tmp/awscli-bundle
+
+- name: Install awscli package.
+  shell: python /tmp/awscli-bundle/install -i /usr/local/aws -b /usr/bin/aws
+  args:
+    creates: /usr/bin/aws
+
+- name: Configure AWS.
+  shell: aws configure set {{ item.key }} {{ item.value }} --profile {{ aws_cli_profile }}
+  no_log: True
```

Comment:
This is a best practice: https://docs.ansible.com/ansible/faq.html#how-do-i-keep-secret-data-in-my-playbook